srml/treasury/src/lib.rs

```diff
@@ -431,11 +432,11 @@ mod tests {
 	type Treasury = Module<Test>;
 
 	fn new_test_ext() -> runtime_io::TestExternalities<Blake2Hasher> {
-		let mut t = system::GenesisConfig::default().build_storage::<Test>().unwrap().0;
-		t.extend(balances::GenesisConfig::<Test>{
+		let mut t = system::GenesisConfig::default().build_storage::<Test>().unwrap();
+		balances::GenesisConfig::<Test>{
 			balances: vec![(0, 100), (1, 99), (2, 1)],
 			vesting: vec![],
-		}.build_storage().unwrap().0);
+		}.build_storage().unwrap().assimilate_storage(&mut t).unwrap();
```

Comment:
```suggestion
		}.assimilate_storage(&mut t).unwrap();
```
the thing is that during its build balance can use what system has put into storage